observability/cockpit/how-to/deactivate-cockpit.mdx

```diff
@@ -0,0 +1,49 @@
+---
+meta: How to deactivate your Cockpit
+  title: How to deactivate your Cockpit
+  description: This page shows you how to deactivate your Cockpit
+content:
+  h1: How to deactivate your Cockpit
+  paragraph: This page shows you how to deactivate your Cockpit
+categories:
+  - observability
+dates:
+  validation: 2022-10-17
+  posted: 2022-10-30
+---
+
+This page shows you how to deactivate your Cockpit.
+
+<Message type="important">
+
+ By deactivating your Cockpit you will no longer be able to monitor your Scaleway resources, configure the sending of external metrics and logs, or receive alerts. All associated data will be permanently deleted.
+
+</Message>
+
+
+<Message type="requirement">
+
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com/)
+  - You have [activated your Cockpit](/observability/cockpit/how-to/activate-cockpit/)
+
+</Message>
+
+1. Log into your [Scaleway Console](https://console.scaleway.com/) account.
```

Comment:
```suggestion
1. Log into your [Scaleway console](https://console.scaleway.com/) account.
```

Reply:
Actually I don't think you need this step because it's already in the requirements. Maybe edit step 2 to be "Click **Cockpit** in the Observability section of the console side menu..."